libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -75,6 +75,7 @@ internal class MapboxTripSession(
             field = value
             if (value == null) {
                 routeAlerts = emptyList()
+                routeProgress = null
```

Comment:
what will happen when the route changes (rather than changes to null).

will the route progress be the old route for a tick?

Reply:
I think that's not currently possible because of https://github.com/mapbox/mapbox-navigation-android/pull/3204 https://github.com/mapbox/mapbox-navigation-android/pull/3424 and https://github.com/mapbox/mapbox-navigation-android/pull/3431